### ISSUE TITLE
Webview height issue

### ISSUE THREAD

**Issue by monish152** [leoiphonedev/ParallaxScroll_IOS#1]
I used library..but webview frame giving issue.

**Comment by leoiphonedev** [comment]
Its not webview its scrollview

**Comment by monish152** [comment]
Yeah I added webview similar like tableview. In tableview height is fine. But webview height is small

**Comment by leoiphonedev** [comment]
Can you please tell me exact issue and what you are trying to achieve as
with few words description its not clear enough to suggest something

**Comment by monish152** [comment]
UIWebView *webView = [[UIWebView alloc] initWithFrame:CGRectMake(0, 0, [UIScreen mainScreen].bounds.size.width, [UIScreen mainScreen].bounds.size.height)];
    webView.delegate = self;
    NSURL *url = [NSURL URLWithString:urlString];
    NSURLRequest *request = [NSURLRequest requestWithURL:url];
    if ([imageName isEqualToString:@"14"]) {
        [webView loadHTMLString:urlString baseURL:nil];
    }
    else{
        [webView loadRequest:request];
    }
    
    
    [self.contentView addSubview:webView];

I added webview like this. And webview height is 150 pixel up.

**Comment by monish152** [comment]
![IMG_4375](https://user-images.githubusercontent.com/1301491/61232135-3b40d080-a74b-11e9-9ab6-2a70d3d74652.PNG)

**Comment by leoiphonedev** [comment]
So you have imge and text, text will scroll upward and image will be fixed

**Comment by monish152** [comment]
no this is webview. text etc loading from web

Everything working fine.

Only webview height is not perfect. You can see red background.

**Comment by leoiphonedev** [comment]
Try to set webview frame inside dispatch async block

**Comment by monish152** [comment]
tried but not working. seems contentview height is not setting properly

**Comment by monish152** [comment]
and most important thing..its working fine on simulator not on device.

**Comment by monish152** [comment]
Seems constraint issue. Contentview height is giving issue
